### PR TITLE
Add 'html' type to 'xhtml'

### PR DESCRIPTION
**Pull request by lcnittl**

### CODE REVIEW

File: identify/extensions.py [hunk 1]
```diff
@@ -241,6 +241,7 @@
     'CMakeLists.txt': EXTENSIONS['cmake'],
     'COPYING': EXTENSIONS['txt'],
     'Dockerfile': {'text', 'dockerfile'},
+    'flake8': EXTENSIONS['ini'] | {'flake8'},
```
Comment (asottile) [inline identify/extensions.py:244]: a file named `flake8` isn't a valid flake8 configuration

Comment (lcnittl) [reply identify/extensions.py:244]: Did I misinterpret [this section](https://flake8.pycqa.org/en/latest/user/configuration.html#user-configuration)?

```Note that Flake8 looks for ~\.flake8 on Windows and ~/.config/flake8 on Linux and other Unix systems.```

Or should it not be included as it is not a typical repo file?

Comment (asottile) [reply identify/extensions.py:244]: yeah those aren't going to be checked in -- also I plan to remove the user configuration at some point -- it causes significant confusion and almost nobody uses it afaik

Comment (lcnittl) [reply identify/extensions.py:244]: I see. I'll remove the commit and force push to my branch.

Comment (lcnittl) [reply identify/extensions.py:244]: Or can you as collaborator drop the commit in the pull request?

Comment (asottile) [reply identify/extensions.py:244]: I could, but it's easier if you do it!

Comment (lcnittl) [reply identify/extensions.py:244]: Alright!

**Pull request merged**